daemon/src/model/cfd.rs

```diff
@@ -700,14 +706,59 @@ pub enum CfdStateChangeEvent {
     CommitTxSent,
 }
 
+/// Returns the Profit/Loss (P/L) as Bitcoin. Losses are capped by the provided margin
 fn calculate_profit(
-    _intial_price: Usd,
-    _current_price: Usd,
-    _interest_per_day: Decimal,
-    _fee: Usd,
-) -> Result<(Amount, Usd)> {
-    // TODO: profit calculation
-    Ok((Amount::ZERO, Usd::ZERO))
+    initial_price: Usd,
+    current_price: Usd,
+    quantity: Usd,
+    margin: Amount,
+) -> Result<(Decimal, Decimal)> {
+    let margin = Decimal::from_f64(margin.as_btc()).context("Expect to be a valid decimal")?;
+
+    let one_through_initial_price = dec!(1)
```

Comment:
Nit: Perhaps `initial_price_inverse` or `one_over_initial_price` is slightly more idiomatic English?